Cargo.toml

```diff
@@ -7,6 +7,8 @@ version = "0.1.0"
 default = ["std", "rocksdb"]
 rocksdb = ["dep:rocksdb"]
 std = ["parity-scale-codec/std", "bitvec/std", "starknet-types-core/std"]
+# internal
+bench = []
```

Comment:
can you add doc about how to use this crate. A README.md

Reply:
you want me to add a doc on how to run these benchmarks?